universal_transfer_operator/example_dags/example_universal_transfer_operator.py

```diff
@@ -0,0 +1,85 @@
+from datetime import datetime
+
+from airflow import DAG
+from datasets.file import File
+from datasets.table import Table
+from universal_transfer_operator import UniversalTransferOperator
+
+from constants import TransferMode
+
+START_DATE = datetime(2000, 1, 1)
+with DAG(
+    "example_universal_transfer_operator",
+    schedule_interval=None,
+    start_date=START_DATE,
+    catchup=False,
+) as dag:
+    transfer_non_native = UniversalTransferOperator(
+        task_id="transfer_non_native",
+        source_dataset=File("gs://uto-test/uto/", conn_id="google_cloud_default", extra={}),
+        destination_dataset=File("s3://astro-sdk-test/uto/", conn_id="aws_default", extra={}),
```

Comment:
How about just `source` and `destination` instead of `source_dataset` and `destination_dataset`?

What do you and others think? @feluelle @utkarsharma2

Reply:
That's a good idea but I wanted to keep the name explicitly clarify user that they need to pass Airflow Dataset URI

Reply:
I am fine either way. Normally, I would prefer `source` and `destination` too, but since the operator itself does not mention that it transfers datasets, I think I like it to be explicit in the args.

Reply:
I think the operator itself does not mention that it transfers datasets, like @feluelle said its better to keep `source_dataset` and `destination_dataset`.